docs/src/topics/grid.md

```diff
@@ -86,7 +86,7 @@ In this space a local numbering of nodes and faces exists, i.e.
 The example shows a local face ID ordering, defined as:
 
 ```julia
-faces(::Lagrange{2,RefCube,1}) = ((1,2), (2,3), (3,4), (4,1))
+faces(::Lagrange{RefQuadrilateral,1}) = ((1,2), (2,3), (3,4), (4,1))
 ```
 
 Other face ID definitions [can be found in the src files](https://github.com/Ferrite-FEM/Ferrite.jl/blob/8224282ab4d67cb523ef342e4a6ceb1716764ada/src/interpolations.jl#L154) in the corresponding `faces` dispatch.
```

Comment:
No, but we do not have docs for these yet. #689 maybe? Also, this doc must be specified further. I just remembered that `faces` does not exist anymore for interpolations (and that this doc string was a bit misleading in first place).

Reply:
> `faces` does not exist anymore for interpolations

What do you mean? On master e.g. `Ferrite.faces(Lagrange{RefQuadrilateral,1}())` works fine?

Reply:
https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/src/deprecations.jl#L8

Reply:
It is really tricky with those deprecations since we don't see anything unless running tests (or explicitly asking for depwarn) - since many users will just run scripts, would it perhaps be better to give an explicit warning?

Sorry for spamming this PR, but just to save the idea in case this could be a valuable pattern, what about 
```julia
function foo(::AbstractFloat)
    println("Allowed")
end
@generated function foo(v::Integer)
    @warn "Calling foo(::Integer) is deprecated"
    return :(println("Was allowed before"))
end
```
Giving 
```
julia> foo(1.)
Allowed

julia> foo(1)
┌ Warning: Calling foo(::Integer) is deprecated
└ @ Main REPL[11]:2
Was allowed before

julia> foo(1)
Was allowed before
```
CC @fredrikekre

Reply:
Okay let us do this specific change in a different PR since I really just want to update the docs here.